solidity/ecdsa/hardhat.config.ts

```diff
@@ -74,7 +74,9 @@ const config: HardhatUserConfig = {
       // we use higher gas price for tests to obtain more realistic results
       // for gas refund tests than when the default hardhat ~1 gwei gas price is
       // used
-      gasPrice: 200000000000, // 200 gwei
+      gasPrice: 200000000000, // 200 gwei,
+      // Ignore contract size on test deployment to hardhat network.
```

Comment:
Let's make it explicit in the comment size of which contracts we are ignoring. We are not interested in ignoring the size of all contracts, just staking (already deployed, problem solved by decreasing the number of the optimizer runs) and test contracts.

Reply:
55a570255c06107468171388011682f71a1d69fc